staging/src/k8s.io/client-go/tools/record/events_cache_test.go

```diff
@@ -228,14 +240,26 @@ func TestEventCorrelator(t *testing.T) {
 			previousEvents:  makeSimilarEvents(defaultAggregateMaxEvents-1, similarEvent, similarEvent.Message),
 			newEvent:        similarEvent,
 			expectedEvent:   setCount(similarEvent, 1),
-			intervalSeconds: defaultAggregateIntervalInSeconds,
+			intervalSeconds: defaultAggregateIntervalInSeconds + 1,
+		},
+		"similar-events-whose-interval-is-defaultAggregateIntervalInSecondsMinusOneSecond-aggregate-event": {
+			previousEvents:  makeSimilarEvents(defaultAggregateMaxEvents-1, similarEvent, similarEvent.Message),
+			newEvent:        similarEvent,
+			expectedEvent:   setCount(aggregateEvent, 1),
+			intervalSeconds: defaultAggregateIntervalInSeconds - 1,
+		},
+		"similar-events-whose-interval-is-defaultAggregateIntervalInSecondsMinusOneSecond-should-count-the-aggregate": {
+			previousEvents:  makeSimilarEvents(defaultAggregateMaxEvents, similarEvent, similarEvent.Message),
+			newEvent:        similarEvent,
+			expectedEvent:   setCount(aggregateEvent, 2),
+			intervalSeconds: defaultAggregateIntervalInSeconds - 1,
```

Comment:
Can we just reuse the "similar-events-should-aggregate-event" and "similar-events-many-times-should-count-the-aggregate", just change their intervalSeconds to `defaultAggregateIntervalInSeconds-1`.